experimental/batch_process/batch_process.py

```diff
@@ -12,9 +15,10 @@
 
 
 def process_folder_of_session_folders(
-        path_to_folder_of_session_folders: Union[str, Path],
-        path_to_camera_calibration_toml: Optional[Union[str, Path]] = None,
-        path_to_blender_executable: Optional[Union[str, Path]] = None,
+    path_to_folder_of_session_folders: Union[str, Path],
+    path_to_camera_calibration_toml: Optional[Union[str, Path]] = None,
```

Comment:
Store paths as strings by default, and convert to Path when you need to. 

This lets us keep path data in a primitive (string) and also lets us get rid of a bunch of "Optionals" in our class/methods definitions.